manifests/server/etcd/setup.pp

```diff
@@ -56,50 +97,61 @@
         ensure => 'absent',
       }
     }
-  } else {
-    package { 'etcd':
+
+    group { $group:
       ensure => $ensure,
-      name   => $package,
+      system => true,
+      gid    => $gid,
     }
-  }
 
-  user { 'etcd':
-    ensure => $ensure,
-  }
-  group { 'etcd':
-    ensure  => $ensure,
-    members => ['etcd'],
+    user { $user:
+      ensure     => $ensure,
+      comment    => 'ETCD user',
+      gid        => $gid,
+      home       => $storage_path,
+      managehome => false,
+      shell      => (fact('os.family') ? {
```

Comment:
selector statement \o/

Reply:
is it good or bad? i copied it over from the main class where the kube-user is created.